### ISSUE TITLE
Upgrade to iOS 10.3 breaks XMPPFramework

### ISSUE THREAD

**Issue by afriedmanGlacier** [ChatSecure/ChatSecure-iOS#728]
I upgraded to iOS 10.3, which forced me to upgrade xcode to 8.3 (and forced me to update OS to 10.12). Now I cannot build because I am getting errors in XMPPFramework that I have not been able to resolve. I looked online and tried several suggestions, but no luck so far. I also started from scratch, completely removing my ChatSecure, cloning from github, and rebuilding, and still go the same errors. Here is one of the 7 errors (they are all the same error but in different Extensions folders of XMPPFramework):

Data Model Compiler - MOMC Error
/Users/afriedman/Documents/ChatSecure-iOS/Submodules/XMPPFramework/Extensions/XEP-0136/CoreDataStorage/XMPPMessageArchiving.xcdatamodeld/XMPPMessageArchiving.xcdatamodel: Cannot find cdtool at '/var/folders/jd/hksw4nmn7wqfmtlpn79gv0gc0000gn/T/AppTranslocation/42D2D252-7CFA-4F3E-9E84-7E4B74C7A0C3/d/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/Library/Xcode/Agents/cdtool': Cannot find a simulator runtime for platform <DVTPlatform:0x7ff841409210:'com.apple.platform.iphonesimulator':<DVTFilePath:0x7ff8414091a0:'/var/folders/jd/hksw4nmn7wqfmtlpn79gv0gc0000gn/T/AppTranslocation/42D2D252-7CFA-4F3E-9E84-7E4B74C7A0C3/d/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform'>>.

**Comment by chrisballinger** [comment]
Do a clean build

Works fine on 8.3 for me

**Comment by afriedmanGlacier** [comment]
Sigh. I've tried that a bunch of times. And I completed blew away ChatSecure and started over. Still the same issue. 

I'm guessing if nobody else has had this issue, then its probably just something in my setup and not a ChatSecure issue. If nobody else runs into this soon, I guess we'll just close it. But hopefully I can figure this out on my end.

I can get it to build if I remove the xcdatamodel files from XMPPFramework, but that causes errors when I run the application. You are on iOS 10.3?

**Comment by afriedmanGlacier** [comment]
Problem fixed. I believe is was related to my upgrade to Xcode or Sierra. I found the fix on 
https://forums.developer.apple.com/thread/48540   about 3/4 of the way down the page. Here's the relevant answer:

"Your issue is the installation issue whereby your install is getting AppTranslocated for some reason.  Some have reported this if they run Xcode-beta.app right out of Downloads, but others have seen it even following our reccomended installation instructions.  We're still investigating in order to provide an update to the release note, but this workaround seems to work for those that we've tried it with:
 
Copy Xcode-beta.app from /Applications to ~/Desktop using Finder
Delete /Applications/Xcode-beta.app
Copy Xcode-beta.app from ~/Desktop to /Applications using finder"

"The bottom line was that when I copied Xcode to Applications folder, there was a bit set on it and that was preventing it from communicating with the simulator."

I wasn't using Xcode beta, but the instructions worked. No issue with ChatSecure.
